### DISCUSSION TITLE
Stacktrace in Pimcore\Model\Version

### DISCUSSION THREAD

**Discussion by quayle** [pimcore/pimcore#16684] [Q&A / Support]
Hello.

In commit: https://github.com/pimcore/pimcore/commit/71edd18477f72c0762f51d18a347a2cc9e51080d there was introduced saving to database stacktrace in version.

Maybe there is something I'm missing, but could someone explain me what is the purpose of always throwing by default some dummy exception..?

https://github.com/pimcore/pimcore/blob/3a8393169f24a4c34f68d64e7ceb39a14346a19b/models/Version.php#L147

Thank you in advance for any answers.

**Comment by blankse** [comment]
It is not necessary to throw the exception to get the stacktrace. I created a PR: https://github.com/pimcore/pimcore/pull/16699

**Reply by quayle** [reply]
Thank you for answer. Yet, why that stacktrace is even generated? What's the purpose of storing it on every save even if it is not thrown..?
